packages/@aws-cdk/aws-lambda-nodejs/parcel/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Based on lambci build image with yarn and parcel
+ARG IMAGE=lambci/lambda:build-nodejs12.x
```

Comment:
Why do we need a default here?

Reply:
We don't but it allows to simply do `docker build .` when developing/testing.

Reply:
Might be slightly misleading. At least add a comment that the CDK will use the correct image based on the runtime selected by the user.